client/stylesheets/_toCat.scss

```diff
@@ -3302,6 +3310,18 @@ On click the images will be displayed at normal size to complete the effect
     float: right;
 } 
 
+.listingItemLocationString {
```

Comment:
Selector `listingItemLocationString` should be written in lowercase with hyphens